### ISSUE TITLE
[3.9.4] pypi wheel package differs from source

### ISSUE THREAD

**Issue by bartekrychlicki** [timothycrosley/isort#220]
Hi! Timothy, I love isort so much but it just gave me a headache and an hour long debug opportunity :)

Simply, https://pypi.python.org/packages/2.7/i/isort/isort-3.9.4-py2.py3-none-any.whl differs to tar.gz and github sources (causing a weird bug that'd remove `# -*- coding: utf-8 -*-` in some of my files). 

How to reproduce:
* unzip the wheel and diff isort/isort.py against the github or PyPI tar.gz version.

I attach the diff as a proof. 

```diff
--- wheeled/isort/isort.py	2014-12-29 16:32:12.000000000 +0100
+++ targzed/isort/isort.py	2014-12-29 22:32:13.000000000 +0100
@@ -106,7 +106,7 @@
         self.number_of_lines = len(self.in_lines)
 
         self.out_lines = []
-        self.comments = {'from': {}, 'straight': {}, 'nested': {}}
+        self.comments = {'from': {}, 'straight': {}, 'nested': {}, 'above': {'straight': {}, 'from': {}}}
         self.imports = {}
         self.as_map = {}
         for section in itertools.chain(SECTIONS, self.config['forced_separate']):
@@ -305,6 +305,9 @@
                 else:
                     import_definition = "import {0}".format(module)
 
+                comments_above = self.comments['above']['straight'].get(module, None)
+                if comments_above:
+                    section_output.append(comments_above)
                 section_output.append(self._add_comments(self.comments['straight'].get(module), import_definition))
 
             from_modules = list(self.imports[section]['from'].keys())
@@ -362,6 +365,9 @@
                             if comment:
                                 single_import_line = self._add_comments(comments, import_start + from_import)
                                 single_import_line += "{0} {1}".format(comments and ";" or "  #", comment)
+                                above_comments = self.comments['above']['from'].get(module, None)
+                                if above_comments:
+                                    section_output.extend(above_comments)
                                 section_output.append(self._wrap(single_import_line))
                                 from_imports.remove(from_import)
                                 comments = None
@@ -400,9 +406,12 @@
                             else:
                                 import_statement = self._wrap(import_statement)
 
+
                     if import_statement:
+                        above_comments = self.comments['above']['from'].get(module, None)
+                        if above_comments:
+                            section_output.extend(above_comments)
                         section_output.append(import_statement)
-
             if section_output:
                 section_name = section
                 if section in SECTIONS:
@@ -669,6 +678,9 @@
                             comments.pop(comments.index(associated_commment))
                     if comments:
                         self.comments['from'].setdefault(import_from, []).extend(comments)
+                    last = self.out_lines and self.out_lines[-1].rstrip() or ""
+                    if last.startswith("#") and not last.endswith('"""') and not last.endswith("'''"):
+                        self.comments['above']['from'].setdefault(import_from, []).append(self.out_lines.pop(-1))
                     if root.get(import_from, False):
                         root[import_from].update(imports)
                     else:
@@ -678,4 +690,7 @@
                         if comments:
                             self.comments['straight'][module] = comments
                             comments = None
+                        last = self.out_lines and self.out_lines[-1].rstrip() or ""
+                        if last.startswith("#") and not last.endswith('"""') and not last.endswith("'''"):
+                            self.comments['above']['from'][module] = self.out_lines.pop(-1)
                         self.imports[self.place_module(module)][import_type].add(module)
```
 
Now the problem itself: Try isort on such file:
```python
# -*- coding: utf-8 -*-
import abc
import string
```
My output:
```diff
$ isort --diff test_file.py 
--- /home/bartek/workspace/x/test_file.py:before
+++ /home/bartek/workspace/x/test_file.py:after
@@ -1,3 +1,3 @@
-# -*- coding: utf-8 -*-
+
 import abc
 import string
```

So the wheel version works fine. Hope that helps!

**Comment by bartekrychlicki** [comment]
I found de68bee9b875ba7e7eb2b614b4bc786f95f8b7f1 now so I understand better now that the wheel works by accident, it's the old version. 

Please, extend the test:
```diff
diff --git a/test_isort.py b/test_isort.py
index 37fd6dc..020d8f0 100644
--- a/test_isort.py
+++ b/test_isort.py
@@ -1112,7 +1112,8 @@ def test_placement_control():

 def test_sticky_comments():
     """Test to ensure it is possible to make comments 'stick' above imports"""
-    test_input = ("import os\n"
+    test_input = ("# something \n"
+                  "import os\n"
                   "\n"
                   "# Used for type-hinting (ref: https://github.com/davidhalter/jedi/issues/414).\n"
                   "from selenium.webdriver.remote.webdriver import WebDriver  # noqa\n")
```

I am sorry but I won't have time to offer a pull request.

**Comment by timothycrosley** [comment]
Thanks for reporting this! I'll try to fix soon and then update the bug. I'm glad you've found isort useful past this issue :)

~Timothy

**Comment by timothycrosley** [comment]
This should be resolved in the latest release 3.9.5, additionally I removed the wheel for 3.9.4 so no one would accidentally install it.

Thanks!

~Timothy Crosley
